### ISSUE TITLE
How to use the Image explainer

### ISSUE THREAD

**Issue by SelComputas** [interpretml/interpret-community#244]
**Describe the bug**

I am unable to understand how to use this program to run my CNN model through it, in order to know which pixels are the essential ones. 

**To Reproduce**
1. Follow your "Getting Started"
2. Open "notebooks/explain-multiclass-classification-local.ipynb"
3. Look at the image;
![image](https://user-images.githubusercontent.com/54527390/82806701-9de41c00-9e86-11ea-9350-d60fc2398854.png)
4. Try to understand how one is supposed to use the Image Kernel mentioned in the image. 

**Expected behavior**
I expect to have a clear and concise way of using your tool in order to obtain some explanation as to what my CNN model is looking at when making predictions. 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version 

**Additional context**
I feel like I am just missing the part of your README-files where you explain how to use the Image Explainer mentioned in the image above.

**Comment by imatiach-msft** [comment]
@SelComputas currently there is an image explainer in the contrib package azureml-contrib-explain-model package but I wouldn't recommend it, as there is no visualization dashboard for it yet.  We have an https://github.com/interpretml/interpret-text repository for text interpretability and logically we would like to add a repository for images eventually, but unfortunately we don't have one yet.  Sorry, we should update that image in the notebooks as it isn't accurate anymore, we open-sourced parts of an old package as this package which has been made an extension to the interpret package.

**Comment by pk2005** [comment]
Hi! Is there any update on this? I have similar issue and wanted to use InterpretML for my CNN. Thanks!

**Comment by imatiach-msft** [comment]
@pk2005 currently we don't have image support yet, sorry.  However, there is a hierarchical image explainer available in SHAP github repository (https://github.com/slundberg/shap) which works well for images.  If you are using pytorch, captum also has great image explainers: https://github.com/pytorch/captum. I'm not sure what is available for tensorflow, I just found this example using integrated gradients:
https://www.tensorflow.org/tutorials/interpretability/integrated_gradients
